cmd/edit/cluster/cmd.go

```diff
@@ -500,6 +518,47 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  AdditionalAllowedPrincipals *******/
+	updateAdditionalAllowedPrincipals := false
+	if additionalAllowedPrincipals != nil {
+		updateAdditionalAllowedPrincipals = true
+	}
+	if !updateAdditionalAllowedPrincipals && additionalAllowedPrincipals == nil &&
+		interactive.Enabled() {
+		updateAdditionalAllowedPrincipalsValue, err := interactive.GetBool(interactive.Input{
+			Question: "Update additional allowed principals",
+			Default:  updateAdditionalAllowedPrincipals,
```

Comment:
Do we need to have an additional question about "Do we want to update?" The below "Additional Allowed Principal ARNs" should be enough though.

Reply:
We do need this because it follows the convention of cluster edit interactive mode by asking "Do you want to update" Y/N before proceeding. The reason is due to this is an optional param so in interactive mode you don't want to ask users for inputs on optional stuffs that they might not want to edit.